opensearch-api/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 ### Added
+- Added create_pit API ([#136](https://github.com/opensearch-project/opensearch-ruby/issues/136))
```

Comment:
I'm wondering if this necessitates multiple entries in the changelog? I would assume that each entry points to a single PR?

Reply:
What's the benefit of splitting it into 6 PRs?

Reply:
Let's merge these lines.